src/wmnds/components/content-tiles/travel-updates/_travel-updates.scss

```diff
@@ -1,3 +1,44 @@
-.wmnds-content-tile-container {
-  width: fit-content;
+.wmnds-content-tile--container {
+  &__wrapper {
+    width: 335px;
+    margin-bottom: $size-sm;
+  }
+
+  &__star {
+    display: block;
+    width: 2rem;
+    height: 2rem;
+    margin-left: 3.5rem;
+    fill: get-color("secondary");
+  }
+
+  &__bus {
+    margin-right: $size-lg;
+    margin-bottom: $size-sm;
+  }
+
+  .wmnds-disruption-indicator-medium--narrow {
+    &-bus {
+      width: 150px;
+      margin-bottom: 10px;
+      padding: 0.7rem;
+
+      .wmnds-disruption-indicator-medium__icon--right {
+        float: right;
+      }
+
+      .wmnds-disruption-indicator-medium__icon--left {
+        margin-right: 25px;
+      }
+    }
+  }
+
+  .wmnds-msg-help {
```

Comment:
This is more acceptable in terms of overwriting a default component, as you aren't doing too much to it.

But in the above example, you had overwritten the component so much that it may be justifiable to create a new component modifier for disruption indicator or remove the overwrite and use as is...and explain to designer that the design doesn't re-use the component correctly.